benchmarks/Riok.Mapperly.Benchmarks/SourceGeneratorBenchmark.cs

```diff
@@ -0,0 +1,80 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Loggers;
+using Microsoft.Build.Locator;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.MSBuild;
+
+namespace Riok.Mapperly.Benchmarks;
+
+[MemoryDiagnoser]
+[InProcess]
+//[EventPipeProfiler(EventPipeProfile.CpuSampling)]
+//[EtwProfiler]
+//[DisassemblyDiagnoser]
+public class SourceGeneratorBenchmark
+{
+    private Compilation? _compilation;
+
+    private CSharpGeneratorDriver? _driver;
+    private MSBuildWorkspace? _workspace;
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        var currentDirectory = Directory.GetCurrentDirectory();
+        if (currentDirectory.Contains("bin", StringComparison.OrdinalIgnoreCase))
+            currentDirectory = currentDirectory.Split("bin")[0];
+        ConsoleLogger.Default.WriteLine("Starting!!! at " + currentDirectory);
+        MSBuildLocator.RegisterDefaults();
+
+        _workspace = MSBuildWorkspace.Create();
+        _workspace.WorkspaceFailed += (sender, args) =>
+        {
+            ConsoleLogger.Default.WriteLineError("-------------------------");
+            ConsoleLogger.Default.WriteLineError(args.Diagnostic.ToString());
+            ConsoleLogger.Default.WriteLineError("-------------------------");
+        };
+
+        var projectFile = Path.Combine(currentDirectory, "../../samples/Riok.Mapperly.Sample/Riok.Mapperly.Sample.csproj");
+        if (!File.Exists(projectFile))
+            throw new Exception("Project doesn't exist");
+        else
+            ConsoleLogger.Default.WriteLine("Project exists!");
+
+        Project? project = null;
+        try
+        {
+            ConsoleLogger.Default.WriteLine("Loading project!!!");
```

Comment:
Nit: I don't think these exclamation marks at the end add any value. These messages don't seem to important to me, that they would need a high attention.